pkg/oauth/user.go

```diff
@@ -207,16 +211,49 @@ func (s *server) Login(c echo.Context) error {
 	return c.NoContent(http.StatusNoContent)
 }
 
+var errInvalidLogoutRedirectURI = errors.DefinePermissionDenied("invalid_logout_redirect_uri", "the redirect URI did not match the one(s) defined in the client")
+var errMissingAccessTokenIDParam = errors.DefinePermissionDenied("missing_param_access_token_id", "Access token ID was not provided")
+
 func (s *server) Logout(c echo.Context) error {
 	ctx := c.Request().Context()
-	session, err := s.getSession(c)
```

Comment:
How about the currenlty active session on the OAuth server? Do we just ignore that now?

Reply:
See my other comment. The access token will be the source for referencing the relevant session now and the auth flows will be initiated by our official (SSO) clients. The oauth server will only handle authentication from now on, without any authenticated views.